src/client/core/utils/is-key-identifier-needed.js

```diff
@@ -0,0 +1,7 @@
+import hammerhead from '../deps/hammerhead';
+
+const browserUtils = hammerhead.utils.browser;
+
+/* eslint-disable no-undef */
+export default IsKeyIdentifierNeeded = () => browserUtils.isSafari || browserUtils.isChrome && browserUtils.version < 54;
```

Comment:
we use upper-cased first letter only for classes
`keyIdentifierRequiredForEvent`